Gemfile

```diff
@@ -8,11 +8,13 @@ gemspec
 gem "rake"
 
 group :development, :test do
+  gem "rails"
```

Comment:
why do we need to include rails gem? I thin the generators are included in railties already.

Reply:
-Yes we need, railties was not enough.